src/main/resources/application.yml

```diff
@@ -97,3 +97,10 @@ rest:
     timeout:
       connect-millis: "\${DEFAULT_REST_TIMEOUT_CONNECT_MILLIS:120000}"
       read-millis: "\${DEFAULT_REST_TIMEOUT_READ_MILLIS:120000}"
+  auth:
+    base-url: "\${AUTH_BASE_URL:http://localhost:8080/auth}"
+    max-attempts: "\${AUTH_MAX_ATTEMPTS:3}" # It's counting also the first attempt
+    wait-time-millis: "\${AUTH_WAIT_TIME_MILLIS:500}"
+    print-body-when-error: "\${AUTH_PRINT_BODY_WHEN_ERROR:true}"
+    post-token:
+      client_secret: "\${AUTH_CLIENT_SECRET:clientsecret}"
```

Comment:
this is not need, instead you have to configure debt-position rest connection (you have to create also a PR on aks-deploy to set the baseUrl